ddtrace/contrib/flask/patch.py

```diff
@@ -58,6 +59,66 @@
 flask_version = parse_version(flask_version_str)
 
 
+class _FlaskWSGIMiddleware(_DDWSGIMiddlewareBase):
+    _request_span_name = "flask.request"
+    _application_span_name = "flask.application"
+    _response_span_name = "flask.response"
+
+    # DEV: We tried using `Flask.finalize_request`, which seemed to work, but gave us hell during tests
+    # DEV: The downside to using `start_response` is we do not have a `Flask.Response` object here,
```

Comment:
This comment is also stale, we no longer patch flask's start_response, the comment still holds up otherwise though